settings.gradle

```diff
@@ -58,6 +58,17 @@ gradleEnterprise {
     }
 }
 
+buildCache {
+    remote(HttpBuildCache) {
```

Comment:
```suggestion
    remote(gradleEnterprise.buildCache) {
```

Reply:
🤦  I must be tired, you were mentioning this line from the start. Thanks!

Reply:
Second run:
PR: https://github.com/jrhee17/armeria/pull/27
diff: https://github.com/line/armeria/compare/main...jrhee17:armeria:main

Reply:
I see, I guess `enabled` meant "enable gradle enterprise build caching" (not "enable build cache overall")
Let me update that if the current setup doesn't work

Reply:
Confirmed that the push is done correctly:
https://ge.armeria.dev/s/nixufg72p3e3c/performance/build-cache